### ISSUE TITLE
Ubuntu sqlx chrono error

### ISSUE THREAD

**Issue by van-sprundel** [loco-rs/loco#852]
**Description**

Running the repo on Ubuntu gives the following error

```rust

    Checking loco-rs v0.10.1 (/home/ramones/git/loco)
error[E0277]: the trait bound `chrono::DateTime<Utc>: sqlx::Encode<'_, sqlx::Postgres>` is not satisfied
   --> src/bgworker/pg.rs:227:11
    |
227 |     .bind(run_at)
    |      ---- ^^^^^^ the trait `sqlx::Encode<'_, sqlx::Postgres>` is not implemented for `chrono::DateTime<Utc>`
    |      |
    |      required by a bound introduced by this call
    |
    = help: the following other types implement trait `sqlx::Encode<'q, DB>`:
              `&T` implements `sqlx::Encode<'q, DB>`
              `&[T]` implements `sqlx::Encode<'q, sqlx::Postgres>`
              `&[u8]` implements `sqlx::Encode<'_, sqlx::Postgres>`
              `&str` implements `sqlx::Encode<'_, sqlx::Postgres>`
              `Box<[u8]>` implements `sqlx::Encode<'_, sqlx::Postgres>`
              `Box<str>` implements `sqlx::Encode<'_, sqlx::Postgres>`
              `Cow<'_, str>` implements `sqlx::Encode<'_, sqlx::Postgres>`
              `Duration` implements `sqlx::Encode<'_, sqlx::Postgres>`
            and 23 others
note: required by a bound in `sqlx::query::Query::<'q, DB, <DB as HasArguments<'q>>::Arguments>::bind`
   --> /home/ramones/.cargo/registry/src/index.crates.io-6f17d22bba15001f/sqlx-core-0.7.4/src/query.rs:81:32
    |
81  |     pub fn bind<T: 'q + Send + Encode<'q, DB> + Type<DB>>(mut self, value: T) -> Self {
    |                                ^^^^^^^^^^^^^^ required by this bound in `Query::<'q, DB, <DB as HasArguments<'q>>::Arguments>::bind`

error[E0277]: the trait bound `chrono::DateTime<Utc>: sqlx::Type<sqlx::Postgres>` is not satisfied
   --> src/bgworker/pg.rs:227:11
    |
227 |     .bind(run_at)
    |      ---- ^^^^^^ the trait `sqlx::Type<sqlx::Postgres>` is not implemented for `chrono::DateTime<Utc>`
    |      |
    |      required by a bound introduced by this call
    |
    = help: the following other types implement trait `sqlx::Type<DB>`:
              `&T` implements `sqlx::Type<DB>`
              `()` implements `sqlx::Type<sqlx::Postgres>`
              `(T1, T2)` implements `sqlx::Type<sqlx::Postgres>`
              `(T1, T2, T3)` implements `sqlx::Type<sqlx::Postgres>`
              `(T1, T2, T3, T4)` implements `sqlx::Type<sqlx::Postgres>`
              `(T1, T2, T3, T4, T5)` implements `sqlx::Type<sqlx::Postgres>`
              `(T1, T2, T3, T4, T5, T6)` implements `sqlx::Type<sqlx::Postgres>`
              `(T1, T2, T3, T4, T5, T6, T7)` implements `sqlx::Type<sqlx::Postgres>`
            and 31 others
note: required by a bound in `sqlx::query::Query::<'q, DB, <DB as HasArguments<'q>>::Arguments>::bind`
   --> /home/ramones/.cargo/registry/src/index.crates.io-6f17d22bba15001f/sqlx-core-0.7.4/src/query.rs:81:49
    |
81  |     pub fn bind<T: 'q + Send + Encode<'q, DB> + Type<DB>>(mut self, value: T) -> Self {
    |                                                 ^^^^^^^^ required by this bound in `Query::<'q, DB, <DB as HasArguments<'q>>::Arguments>::bind`

error[E0277]: the trait bound `chrono::DateTime<Utc>: sqlx::Decode<'_, sqlx::Postgres>` is not satisfied
   --> src/bgworker/pg.rs:247:21
    |
247 |         run_at: row.get("run_at"),
    |                     ^^^ the trait `sqlx::Decode<'_, sqlx::Postgres>` is not implemented for `chrono::DateTime<Utc>`
    |
    = help: the following other types implement trait `sqlx::Decode<'r, DB>`:
              `&'r JsonRawValue` implements `sqlx::Decode<'r, DB>`
              `&'r [u8]` implements `sqlx::Decode<'r, sqlx::Postgres>`
              `&'r str` implements `sqlx::Decode<'r, sqlx::Postgres>`
              `()` implements `sqlx::Decode<'r, sqlx::Postgres>`
              `(T1, T2)` implements `sqlx::Decode<'r, sqlx::Postgres>`
              `(T1, T2, T3)` implements `sqlx::Decode<'r, sqlx::Postgres>`
              `(T1, T2, T3, T4)` implements `sqlx::Decode<'r, sqlx::Postgres>`
              `(T1, T2, T3, T4, T5)` implements `sqlx::Decode<'r, sqlx::Postgres>`
            and 31 others
note: required by a bound in `sqlx::Row::get`
   --> /home/ramones/.cargo/registry/src/index.crates.io-6f17d22bba15001f/sqlx-core-0.7.4/src/row.rs:70:12
    |
67  |     fn get<'r, T, I>(&'r self, index: I) -> T
    |        --- required by a bound in this associated function
...
70  |         T: Decode<'r, Self::Database> + Type<Self::Database>,
    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^ required by this bound in `Row::get`

error[E0277]: the trait bound `chrono::DateTime<Utc>: sqlx::Type<sqlx::Postgres>` is not satisfied
   --> src/bgworker/pg.rs:247:21
    |
247 |         run_at: row.get("run_at"),
    |                     ^^^ the trait `sqlx::Type<sqlx::Postgres>` is not implemented for `chrono::DateTime<Utc>`
    |
    = help: the following other types implement trait `sqlx::Type<DB>`:
              `&T` implements `sqlx::Type<DB>`
              `()` implements `sqlx::Type<sqlx::Postgres>`
              `(T1, T2)` implements `sqlx::Type<sqlx::Postgres>`
              `(T1, T2, T3)` implements `sqlx::Type<sqlx::Postgres>`
              `(T1, T2, T3, T4)` implements `sqlx::Type<sqlx::Postgres>`
              `(T1, T2, T3, T4, T5)` implements `sqlx::Type<sqlx::Postgres>`
              `(T1, T2, T3, T4, T5, T6)` implements `sqlx::Type<sqlx::Postgres>`
              `(T1, T2, T3, T4, T5, T6, T7)` implements `sqlx::Type<sqlx::Postgres>`
            and 31 others
note: required by a bound in `sqlx::Row::get`
   --> /home/ramones/.cargo/registry/src/index.crates.io-6f17d22bba15001f/sqlx-core-0.7.4/src/row.rs:70:41
    |
67  |     fn get<'r, T, I>(&'r self, index: I) -> T
    |        --- required by a bound in this associated function
...
70  |         T: Decode<'r, Self::Database> + Type<Self::Database>,
    |                                         ^^^^^^^^^^^^^^^^^^^^ required by this bound in `Row::get`

error[E0277]: the trait bound `chrono::DateTime<Utc>: sqlx::Encode<'_, _>` is not satisfied
   --> src/bgworker/pg.rs:276:15
    |
276 |         .bind(next_run_at)
    |          ---- ^^^^^^^^^^^ the trait `sqlx::Encode<'_, _>` is not implemented for `chrono::DateTime<Utc>`
    |          |
    |          required by a bound introduced by this call
    |
    = help: the following other types implement trait `sqlx::Encode<'q, DB>`:
              `&T` implements `sqlx::Encode<'q, DB>`
              `&[T]` implements `sqlx::Encode<'q, sqlx::Postgres>`
              `&[u8]` implements `sqlx::Encode<'_, sqlx::Postgres>`
              `&str` implements `sqlx::Encode<'_, sqlx::Postgres>`
              `Box<[u8]>` implements `sqlx::Encode<'_, sqlx::Postgres>`
              `Box<str>` implements `sqlx::Encode<'_, sqlx::Postgres>`
              `Cow<'_, str>` implements `sqlx::Encode<'_, sqlx::Postgres>`
              `Duration` implements `sqlx::Encode<'_, sqlx::Postgres>`
            and 23 others
note: required by a bound in `sqlx::query::Query::<'q, DB, <DB as HasArguments<'q>>::Arguments>::bind`
   --> /home/ramones/.cargo/registry/src/index.crates.io-6f17d22bba15001f/sqlx-core-0.7.4/src/query.rs:81:32
    |
81  |     pub fn bind<T: 'q + Send + Encode<'q, DB> + Type<DB>>(mut self, value: T) -> Self {
    |                                ^^^^^^^^^^^^^^ required by this bound in `Query::<'q, DB, <DB as HasArguments<'q>>::Arguments>::bind`

error[E0277]: the trait bound `chrono::DateTime<Utc>: sqlx::Type<_>` is not satisfied
   --> src/bgworker/pg.rs:276:15
    |
276 |         .bind(next_run_at)
    |          ---- ^^^^^^^^^^^ the trait `sqlx::Type<_>` is not implemented for `chrono::DateTime<Utc>`
    |          |
    |          required by a bound introduced by this call
    |
    = help: the following other types implement trait `sqlx::Type<DB>`:
              `&T` implements `sqlx::Type<DB>`
              `()` implements `sqlx::Type<sqlx::Postgres>`
              `(T1, T2)` implements `sqlx::Type<sqlx::Postgres>`
              `(T1, T2, T3)` implements `sqlx::Type<sqlx::Postgres>`
              `(T1, T2, T3, T4)` implements `sqlx::Type<sqlx::Postgres>`
              `(T1, T2, T3, T4, T5)` implements `sqlx::Type<sqlx::Postgres>`
              `(T1, T2, T3, T4, T5, T6)` implements `sqlx::Type<sqlx::Postgres>`
              `(T1, T2, T3, T4, T5, T6, T7)` implements `sqlx::Type<sqlx::Postgres>`
            and 31 others
note: required by a bound in `sqlx::query::Query::<'q, DB, <DB as HasArguments<'q>>::Arguments>::bind`
   --> /home/ramones/.cargo/registry/src/index.crates.io-6f17d22bba15001f/sqlx-core-0.7.4/src/query.rs:81:49
    |
81  |     pub fn bind<T: 'q + Send + Encode<'q, DB> + Type<DB>>(mut self, value: T) -> Self {
    |                                                 ^^^^^^^^ required by this bound in `Query::<'q, DB, <DB as HasArguments<'q>>::Arguments>::bind`

For more information about this error, try `rustc --explain E0277`.
error: could not compile `loco-rs` (lib) due to 6 previous errors
```

**To Reproduce**

Run `cargo check` in master on Ubuntu (in WSL).

**Expected Behavior**

`cargo check` to succeed.

**Environment:**

OS: Ubuntu 24.04.1 LTS on Windows 10 x86_64
Kernel: 5.15.153.1-microsoft-standard-WSL2

rustup 1.27.1 (54dd3d00f 2024-04-24)
info: This is the version for the rustup toolchain manager, not the rustc compiler.
info: The currently active `rustc` version is `rustc 1.81.0 (eeb90cda1 2024-09-04)`

**Additional Context**

Adding the `chrono` feature and bumping the version seems to fix this for me.
```toml
sqlx = { version = "0.8", default-features = false, features = [
    "postgres",
    "chrono"
], optional = true }
```

**Comment by AndreiOrmanji** [comment]
@van-sprundel,
I think your project is using shuttle crates. If you set all versions of shuttle crates to 0.47, the issue will disappear.

**Comment by Feriixu** [comment]
I can reproduce.
The bug seems to have been introduced in `4f204a5`.

**Comment by jondot** [comment]
fixed on 0.11.0
